external-crates/move/crates/move-core-types/src/parsing/parser.rs

```diff
@@ -447,20 +446,22 @@ pub fn parse_u256(s: &str) -> Result<(U256, NumberFormat), U256FromStrError> {
 }
 
 // Parse an address from a decimal or hex encoding
-pub fn parse_address_number(s: &str) -> Option<([u8; AccountAddress::LENGTH], NumberFormat)> {
+pub fn parse_address_number(s: &str) -> Option<(AccountAddress, NumberFormat)> {
     let (txt, base) = determine_num_text_and_base(s);
-    let parsed = BigUint::parse_bytes(
-        txt.as_bytes(),
+    let max_len = match base {
+        NumberFormat::Hex => AccountAddress::LENGTH * 2,
+        NumberFormat::Decimal => 78,
```

Comment:
Where is the 78 coming from? Is it possible to express that in terms of `AccountAddress::LENGTH`?

Reply:
Yea! I can do that. It's based off of this: https://stackoverflow.com/questions/43787672/the-max-number-of-digits-in-an-int-based-on-number-of-bits

So would be `241 * AccountAddress::LENGTH / 100 + 1

Reply:
Wowza. Yeah I would definitely comment the shit out of that number lol

Reply:
Updated it to be a const + I put a link to the stack overflow article on it as that did a much better job than any explanation I can give.

Reply:
I'm reconsidering this move to u256 here altogether -- in particular `_`s in addresses are going to mess this change all up. So will need to look a bit more at it and will land outside of this stack.